docs/docs-content/enterprise-version/install-palette/install-on-kubernetes/uninstall.md

```diff
@@ -0,0 +1,115 @@
+
+- Helm is installed and available on your host machine.
+
+- kubectl is installed and available on your host machine.
+
+## Uninstall Palette
+
+1. Ensure you are using the appropriate context where Palette is deployed.
+
+   ```shell
+   kubectl config current-context
+   ```
+
+2. Issue the following command to start uninstalling Reach. This will remove all resources related to Reach that are
+   managed by Helm. However, some resources created by Helm hooks are not managed by helm and will require additional
+   manual intervention to remove.
+
+   ```shell
+   helm uninstall reach-system
+   ```
+
+3. Issue the following commands to remove the remaining Reach system resources.
+
+   ```shell
+   kubectl delete ns reach-system
+   kubectl delete crd podpresets.reach.spectrocloud.com clusterpodpresets.reach.spectrocloud.com
+   kubectl delete mutatingwebhookconfiguration reach-mutating-webhook-configuration
+   kubectl delete validatingwebhookconfiguration reach-validating-webhook-configuration
+   kubectl delete clusterrolebinding reach-manager-rolebinding
+   kubectl delete clusterrolebinding reach-proxy-rolebinding
+   kubectl delete clusterrole reach-manager-role
+   kubectl delete clusterrole reach-metrics-reader
+   kubectl delete clusterrole reach-proxy-role
+   ```
+
+4. Issue the following command to start uninstalling the Palette management plane. Similar to Reach, this will only
+   remove the resources managed by Helm and the remaining resources will require additional manual intervention.
+
+   ```shell
+   helm uninstall hubble
+   ```
+
+5. Issue the following command to remove the namespace and custom resource definitions related to Palette management
+   plane.
+
+   ```shell
+   kubectl delete ns hubble-system || kubectl delete crd spectroclusteractions.jet.cluster.spectrocloud.com
+   ```
+
+6. Issue the following command to remove the `image-swap` chart.
+
+   ```shell
+   helm uninstall image-swap
+   ```
+
+7. Issue the following commands to remove the remaining resources related to `image-swap`.
+
+   ```shell
+   kubectl delete ns imageswap-system
+   kubectl delete mutatingwebhookconfiguration imageswap-webhook
+   kubectl delete csr imageswap.imageswap-system.cert-request
+   ```
+
+8. Issue the following command to uninstall Cert Manager. Cert Manager does not reply on any Helm hooks and the Helm
+   uninstall command will uninstall all related resources.
+
+   ```shell
+   helm uninstall cert-manager
+   ```
+
+## Validate
+
+1. Connect to your cluster via kubectl.
+
+2. Issue the following command to get a list of remaining namespaces. Confirm that all Palette-related namespaces have
+   been deleted.
+
+   ```shell
+   kubectl get namespaces
+   ```
+
+3. Issue the following command to retrieve all resources in your cluster across all namespaces. Confirm that no
+   Palette-related resources are remaining.
+
+   ```shell
+   kubectl get all --all-namespaces
+   ```
```

Comment:
Another verification step would be to issue `helm ls` and ensure the chart is not installed.